frontend/src/App.vue

```diff
@@ -82,11 +82,14 @@
                       >
                         {{ menuElem.displayName }}
                       </a>
-                      <div class="navbar-dropdown has-background-primary-lighter p-0">
+                      <div
+                        class="navbar-dropdown has-background-primary-lighter p-0"
+                        :class="{ ' is-right': menuElem.subMenuRight }"
```

Comment:
is the extra space in`' is-right'` there for a reason or typo?

Reply:
I think it is needed because it gets (optionally) appended to the classes in the line above, or else it becomes `p-0is-right`.